experiment/simulation/index.html

```diff
@@ -1,43 +1,185 @@
+                                    <option value="Dodecahedron">Dodecahedron</option>
+                                    <option value="Cube">Cube</option>
+                                </select>
+                                    </div>
+                                </div>
+                                </br>
+                                </br>
+
+                                <input class="input " style="width: 200px; float:left" id="x" type="number" placeholder="X" step=".01" value="0">
+                                <input class="input " style="width: 200px; float:left" id="y" type="number" placeholder="Y" step=".01" value="0">
+                                <input class="input " style="width: 200px; float:left" id="z" type="number" placeholder="Z" step=".01" value="0">
+
+                                <div class="control">
+                                    <button class="buttonisprimary" style="width: 200px; float:left; margin: 1rem;">Edit</button>
+                                </div>
+                            </p>
+                        </div>
+
+                    </div>
+
+                    <div id="add-modal" class="modal is-responsive">
+                        <div class="modal-content">
+                            <span class="close">&times;</span>
+                            <p>
+                                <p class="v-datalist-title"> Add Shape Menu</p>
+                                <div id="ShapeAdd" class="control">
+                                    <div class="select" style="float:left;">
+                                        <select name="" id="shape-add-dropdown" value="dropdown">
+                                            <option value="Tetrahedron">Tetrahedron</option>
+                                            <option value="Octahedron">Octahedron</option>
+                                            <option value="Dodecahedron">Dodecahedron</option>
+                                            <option value="Cube">Cube</option>
+                                </select>
+                                    </div>
+                                </div>
+                                </br>
+                                </br>
+
+                                <input class="input " style="width: 100px; float:left; margin: 0.25rem;" id="x1" type="number" placeholder="X" step=".01" value="0">
+                                <input class="input " style="width: 100px; float:left;margin: 0.25rem;" id="y1" type="number" placeholder="Y" step=".01" value="0">
+                                <input class="input " style="width: 100px; float:left;margin: 0.25rem;" id="z1" type="number" placeholder="Z" step=".01" value="0">
+                                <div class="control">
+                                    <button class="buttonissecondary" id="modalBox_addsubmit" style="width: 10rem; float:left; margin: 1rem;">Add!</button>
```

Comment:
Please use hyphens to separate words in a multi-word-id

Reply:
fixed